webServer.js

```diff
@@ -25,7 +25,7 @@ throng({
   lifetime: Infinity
 }, start);
 
-function start() {
+function start () {
```

Comment:
I believe we currently only add a space on anonymous functions, is this something we can configure in eslint?

cc @PeterDaveHello

Reply:
Yup - i've modified it to `["error", {"anonymous": "always", "named": "never", "asyncArrow": "always"}]`